tests/integration/test_pagure.py

```diff
@@ -501,6 +503,8 @@ def test_pr_status(self):
         assert statuses
         assert len(statuses) >= 0
         assert statuses[-1].state == CommitStatus.success
+        assert statuses[-1].created == datetime(year=2019, month=12, day=2, hour=14, minute=16, second=11)
```

Comment:
Probably a typo:

```suggestion
        assert statuses[-1].created == datetime(year=2019, month=12, day=2, hour=13, minute=16, second=11)
```

See [the output](https://softwarefactory-project.io/zuul/t/local/build/f2589ccd20df4cec98bf27d3aaac2fe0/console) from zuul, our CI system:
```
>       assert statuses[-1].created == datetime(year=2019, month=12, day=2, hour=14, minute=16, second=11)
E       AssertionError: assert datetime.datetime(2019, 12, 2, 13, 16, 11) == datetime.datetime(2019, 12, 2, 14, 16, 11)
E        +  where datetime.datetime(2019, 12, 2, 13, 16, 11) = <ogr.services.pagure.flag.PagureCommitFlag object at 0x7f8d42b21b90>.created
E        +  and   datetime.datetime(2019, 12, 2, 14, 16, 11) = datetime(year=2019, month=12, day=2, hour=14, minute=16, second=11)
```

(You can get to logs via links commented by zuul and selecting the `Console` tab on the top of the page.)

Reply:
Thanks for feedback. I'll start working on it right away.